codex_of_the_damned/templates/archetypes/experiments/creeping-sabotage.html

```diff
@@ -0,0 +1,78 @@
+{% extends "archetypes/_layout.html" %}
+{% import "archetypes/experiments/creeping-sabotage.json" as deck_json %}
+
+{% block meta %}
+<meta property="decklist" content='{{ deck_json | tojson }}' />
+{% endblock %}
+
+{% block archetype_name %}
+<h1><img src="{{ url_for('static', filename='img/icons/clan-nosferatu.svg') }}"> Creeping Sabotage</h1>
+
+<div class="warning">
+    {% trans trimmed
+        creeping_sabotage=card("Creeping Sabotage"),
+		github=external("https://github.com/lionel-panhaleux/codex-of-the-damned", "Github")
+    %}
+    This is an experimental build around a card release on novembre 2020: {{ creeping_sabotage }}.
+	The decklist proposed here is subject to change before reaching its highest level of
+	performance. Feel free to contact us on our {{ github }} to help us better the concept.
+    {% endtrans %}
+</div>
+{% endblock %}
+
+{% block archetype %}
+{% trans trimmed
+	deep_song=card("Deep Song"),
+	creeping_sabotage=card("Creeping Sabotage"),
+	bums_rush=card("Bum's Rush"),
+	the_warrens=card("The Warrens"),
+
+	combat=link("/strategy/combat", _("Combat")),
+	actions=link("/strategy/fundamentals", _("Actions"), _anchor="AB"),
+	stealth=link("/strategy/fundamentals", _("Stealth"), _anchor="stealth-and-intercept"),
+	range_management=link("/strategy/combat", _("Range management"), _anchor="range-management"),
+
+	nananimalism=link("/archetypes/nananimalism", "Nananimalism"),
+	euro_brujah=link("/archetypes/euro-brujah", "Euro Brujah"),
+	modules=link("/strategy/deck-building", _("Modules"), _anchor="modules-and-density")
+%}
+<h2>Highlights</h2>
+<p>
+	This is a <em>rush</em> deck based on mid-cap {{ POT }} vampires.
+</p>
+<p>
+	Ordinary rush deck uses {{ ANI }} as the key discipline because {{ deep_song }}
+	is both the means to rush and the means to oust the prey. There is no alternative to
+	{{ deep_song }} for {{ POT }} decks. This means that the deck needs a means to rush
+	and a means to oust the prey.
```

Comment:
```suggestion
	Ordinary rush deck uses {{ ANI }} as the key discipline because {{ deep_song }}
	is both the means to rush and the means to oust the prey. There is no alternative to
	{{ deep_song }} for {{ POT }} decks. This means that the deck needs a means to rush, 
	{{ bums_rush }}, and a means to oust the prey: {{ creeping_sabotage }}. 
```